src/lusty/filesystem-explorer.rb

```diff
@@ -184,6 +188,31 @@ def all_files_at_view
       end
     end
 
+    def all_remote_files(view, view_str)
+      host, path = view_str[6..-1].split('/', 2)
+      option = if LustyM::option_set?("AlwaysShowDotFiles") or \
+         current_abbreviation()[0] == ?.
+                'a'
+               else
+                 ''
+               end
+
+      files_s = `ssh #{host} ls -1FL#{option} -- #{path}`
+      files = files_s.split("\n")
```

Comment:
You could combine this with the previous line so that there's no `files_s` variable.